src/Tribe/Main.php

```diff
@@ -1631,9 +1631,14 @@ public function displayEventVenueDropdown( $post_id ) {
 					Tribe__Events__Linked_Posts::instance()->saved_linked_post_dropdown( Tribe__Events__Venue::POSTTYPE, $venue_id );
 					$venue_pto = get_post_type_object( self::VENUE_POST_TYPE );
 					if ( current_user_can( $venue_pto->cap->edit_posts ) ) {
+						//Use Admin Link Unless on Community Events Editor then use Front End Link to Edit
+						$edit_link = admin_url( sprintf( 'post.php?action=edit&post=%s', $venue_id ) );
+						if ( tribe_is_community_edit_event_page() ) {
```

Comment:
I find it so weird that this method is in the-events-calendar and not events-community. Just a comment on life in general - not about the state of the code in this PR :)